core/src/main/java/org/openstack4j/model/network/ext/LbPoolV2.java

```diff
@@ -61,6 +61,10 @@
      */
     List<ListItem> getListeners();
 
+    /**
+     * @return The ID of the load balancer under which this pool will be created.
+     */
+    String getLoadbalancerId();
```

Comment:
Prefer spell it `LoadBalancer` (capital `B`) for consistency with the rest of the code base.